pkg/controller/datasource-controller.go

```diff
@@ -215,7 +248,26 @@ func addDataSourceControllerWatches(mgr manager.Manager, c controller.Controller
 		predicate.Funcs{
 			CreateFunc: func(e event.CreateEvent) bool { return true },
 			DeleteFunc: func(e event.DeleteEvent) bool { return true },
-			UpdateFunc: func(e event.UpdateEvent) bool { return false },
+			UpdateFunc: func(e event.UpdateEvent) bool {
+				pvcOld, okOld := e.ObjectOld.(*corev1.PersistentVolumeClaim)
+				pvcNew, okNew := e.ObjectNew.(*corev1.PersistentVolumeClaim)
+				return okOld && okNew && pvcOld.Status.Phase != pvcNew.Status.Phase
```

Comment:
IsPopulated maybe?

Reply:
not sure you are interested in any phase change even if it's to `Bound`.
afaiu, you only care about `if IsPopulated(pvcNew) && !IsPopulated(pvcOld)`?

Reply:
I think we may want just "false" here since the IsPopulated changes are reflected on the DV update predicate

Reply:
actually, change from Pending to Bound may be interesting, that is how populator PVCs signal their work is done
same for restore from snap/pvc etc

Reply:
ok, cool

Reply:
yep I think we should keep this